Add sampling info to Tracing tasks (#3397)
istio/istio.io: content/docs/tasks/telemetry/distributed-tracing/overview/index.md

* `x-b3-flags`
* `x-ot-span-context`

> If you look in the sample services, you can see that the `productpage` service (Python) extracts the required headers from an HTTP request using OpenTracing libraries:

Reviewer: URL to OpenTracing perhaps?

Author: Done.

> If you look in the sample services, you can see that the `productpage` service (Python) extracts the required headers from an HTTP request using [OpenTracing](https://opentracing.io/) libraries:

Reviewer 2: The original had some of the issues I fix below.
```suggestion
If you look at the Python `productpage` service, for example, you see that the service extracts the required headers from an HTTP request using [OpenTracing](https://opentracing.io/) libraries:
```

Author: done.

> If you look at the sample Python `productpage` service, for example, you see that the application extracts the required headers from an HTTP request using [OpenTracing](https://opentracing.io/) libraries:

Reviewer 2: ```suggestion
If you look at the sample Python `productpage` service, for example, 
you see that the application extracts the required headers from an HTTP request 
using the [OpenTracing](https://opentracing.io/) libraries:
```
